File: C#/Projects/SnomedQuery.Parser/RF2ItemSingle.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SnomedQuery.Parser
{
  /// <summary>
  /// Base class for RF@ single item classes.
  /// </summary>
  public class RF2ItemSingle : RF2Item
  {
    /// <summary>
    /// Snomed effective time that item is active
    /// </summary>
    public DateTime EffectiveTime;

    /// <summary>
    /// Snomed active field.
    /// </summary>
    public bool Active;

    /// <summary>
    /// Snomed module id
    /// </summary>
    public Int64 ModuleId;
  }
}

File: Java/output/SnomedQuery/Parser/RF2ItemSingle.java
package SnomedQuery.Parser;

  /** 
   Base class for RF@ single item classes.
  */
  public class RF2ItemSingle extends RF2Item
  {
	/** 
	 Snomed effective time that item is active
	*/
	public java.time.LocalDateTime EffectiveTime = java.time.LocalDateTime.MIN;

	/** 
	 Snomed active field.
	*/
	public boolean Active;

	/** 
	 Snomed module id
	*/
	public long ModuleId;
  }